v22.2/alter-backup-schedule.md

```diff
@@ -0,0 +1,190 @@
+`FULL BACKUP crontab / ALWAYS` | Specify when to take a new full backup. Define the schedule as a `STRING` in [crontab format](https://en.wikipedia.org/wiki/Cron) or as `ALWAYS`. <br><br>`FULL BACKUP ALWAYS` will trigger `RECURRING` to always take full backups. **Note:** If you do not have an Enterprise license then `ALWAYS` is the only accepted value of `FULL BACKUP`. <br><br>If you omit the `FULL BACKUP` clause, the default backup schedule will be as follows: <ul><li>`RECURRING` <= 1 hour: Default to `FULL BACKUP '@daily'`</li><li>`RECURRING` <= 1 day: Default to `FULL BACKUP '@weekly'`</li><li>Otherwise: Default to `FULL BACKUP ALWAYS`</li></ul>
+`schedule_option` | Control the schedule behavior with a comma-separated list of these [schedule options](#schedule-options).
+
+### Backup options
+
+You can use the backup options in this table to control the behavior of your backups. See [Apply different options to scheduled backups](#apply-different-options-to-scheduled-backups) for an example.
+
+{% include {{ page.version.version }}/backups/backup-options-for-schedules.md %}
+
+### Schedule options
+
+{{site.data.alerts.callout_danger}}
+**This feature is in preview.**  Its interface, options, and outputs are subject to change, and there may be bugs.
+
+If you encounter a bug, please [file an issue](file-an-issue.html).
+{{site.data.alerts.end}}
+
+You can use the schedule options in this table to control the behavior of your backup schedule. See [Apply different options to scheduled backups](#apply-different-options-to-scheduled-backups) for an example.
+
+{% include {{ page.version.version }}/backups/schedule-options.md %}
+
+## Examples
+
+The examples in this section start with the following created backup schedule. Each section follows on from the previous example's schedule state.
+
+First, create a schedule that will take daily full backups of the cluster: 
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE SCHEDULE aws_backups
+  FOR BACKUP INTO 'external://s3_storage'
+    RECURRING '@daily'
+    FULL BACKUP ALWAYS
+    WITH SCHEDULE OPTIONS first_run = 'now', ignore_existing_backups;
+~~~
+
+This statement specifies:
+
+- `'external://s3_storage'`: Use the storage location represented by this [external connection](create-external-connection.html) URI.
+- `first_run = 'now'`: Take the first full backup immediately rather than wait for its next `RECURRING` time.
+- `ignore_existing_backups`: Ignore any existing backups already present in the storage location.
+
+The output shows that the [`detached` option](#detached) is implicitly added to scheduled backups:
```

Comment:
nit: the output indicates a successful execution, first and foremost. Recommend a slight switcharound:

> The command returns the following output. Note that the [`detached` option](#detached) has been implicitly added because this backup has been configured to run on a schedule.